mobile/src/main/scala/com/github/shadowsocks/bg/Executable.scala

```diff
@@ -18,10 +21,11 @@ object Executable {
 
   val EXECUTABLES = Array(SS_LOCAL, SS_TUNNEL, PDNSD, REDSOCKS, TUN2SOCKS, OVERTURE)
 
-  def killAll() {
-    val killer = new ProcessBuilder("killall" +: EXECUTABLES: _*).start()
-    if (!killer.waitFor(1, TimeUnit.SECONDS))
-      Log.w("killall", "%s didn't exit within 1s. Post-crash clean-up may have failed."
-        .formatLocal(Locale.ENGLISH, killer.toString))
+  def killAll(context: Context) {
+    val manager = context.getSystemService(Context.ACTIVITY_SERVICE).asInstanceOf[ActivityManager]
```

Comment:
I don't see a reason to use context here. Just use global app object defined in ShadowsocksApplication.